qiskit_ibm_runtime/api/rest/utils/data_mapper.py

```diff
@@ -73,3 +73,66 @@ def to_python_identifier(name: str) -> str:
         name += "_"
 
     return name
+
+
+def map_job_response(data: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
Are these mapper functions actually used? I thought they are only for old IQP jobs.

Reply:
Nope, not used, I've removed them